op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```diff
@@ -0,0 +1,217 @@
+# Runbook: Incident Response for Faultproof Withdrawals
+- [Runbook](#runbook)
```

Comment:
This table of contents looks out of date, I would suggest autogenerating it with doctoc like we do in the monorepo